src/samples/server/Elsa.Samples.Server.Host/Startup.cs

```diff
@@ -102,10 +107,16 @@ public void Configure(IApplicationBuilder app)
             }
 
             app
-                .UseCors()
+                //.UseCors()
+                .UseCors("ClientPermission")
                 .UseElsaFeatures()
                 .UseRouting()
-                .UseEndpoints(endpoints => { endpoints.MapControllers(); });
+                .UseEndpoints(endpoints => { 
+                    endpoints.MapControllers();
+                    endpoints.MapHub<WorkflowTestHub>("/hubs/workflowTest");
```

Comment:
It would be good to provide this as an extension method from the `Elsa.Server.Api` project called e.g. `MapWorkflowTestHub()`. Might be easier to discover for users and allows us to change the path in the future.